.github/workflows/docs.yml

```diff
@@ -3,7 +3,8 @@ name: Build sphinx docs
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["main"]
```

Comment:
I think we want this to run always, but make the very last step (upload artifact) only run on main via this: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif

This way, we can ensure the Pages are getting built and at least not failing before it hits main.

Reply:
Good point, I'll work on this.